main_test.go

```diff
@@ -48,9 +48,9 @@ func TestCLI(t *testing.T) {
 	c.Run(t, TestCase{
 		Command: "list",
 		StdOut: `
-NAME          ADDRESS                     NAMESPACE    STATUS    
-localhost     localhost:7233              default      
-production    temporal.example.com:443    myapp        active`,
+NAME          ADDRESS                     NAMESPACE    WEB    STATUS    
+localhost     localhost:7233              default             
```

Comment:
Mind adding a web address to one of these contexts so we can test that output as well?

Reply:
Yep, done